client/src/main/scala/drt/client/components/TerminalDesksAndQueues.scala

```diff
@@ -154,6 +186,15 @@ object TerminalDesksAndQueues {
           )).toTagMod
       }
 
+
+      def viewWaitTimeControls: TagMod = {
+        List(
+          <.div(^.className := s"selector-control view-type-control $viewRecsClass",
+            <.input.checkbox(^.checked := state.showWaitColumn, ^.onChange ==> toggleWaitColumn, ^.id := "toggle-showWaitingTime"),
+            <.label(^.`for` := "show-waitTime", "Toggle Wait Time")
```

Comment:
this `for` just needs its value changing to `toggle-showWaitingTime` to match the id of the checkbox

Reply:
Let's change the label value to read "Display wait times" to make it more intuitive